### PR TITLE
chore: update reference creation dialouges

### PR DESCRIPTION
**Pull request by raastinn**

### CODE REVIEW

File: src/js/references/components/CreateReference.tsx [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
         <ContainerNarrow>
             <ViewHeader title="Create Reference">
                 <ViewHeaderTitle>Create Reference</ViewHeaderTitle>
             </ViewHeader>
             <Tabs>
-                <TabsLink
-                    to={{ state: { emptyReference: true } }}
-                    isActive={(_, location) => location.state.emptyReference}
-                >
+                <TabsLink to={{ state: { emptyReference: true } }} isActive={() => location.state.emptyReference}>
```
Comment (ReeceHoffmann) [inline src/js/references/components/CreateReference.tsx:24]:
```suggestion
                <TabsLink to={{ state: { emptyReference: true } }} isActive={location.state.emptyReference}>
```
Can you just exclude the arrow functions? Seems like the kind of thing you should be able to do, but depends on TabsLink api.

File: src/tests/fake/references.ts [hunk 2]
```diff
@@ -125,3 +126,23 @@ export function mockApiCloneReference(name: string, description: string, referen
...
+ */
+export function mockApiCreateReference(
+    name: string,
+    description: string,
+    data_type: ReferenceDataType,
+    organism: string,
+) {
+    const reference = createFakeReference({ name, description, data_type, organism });
+
+    return nock("http://localhost").post("/api/refs", { name, description, data_type, organism }).reply(201, reference);
```
Comment (ReeceHoffmann) [inline src/tests/fake/references.ts:147]: Does the API return the entire ref? Your typing with the mutation suggests your only getting some value back, but here your returning the entire reference. Update whichever one is incorrect

**Review by ReeceHoffmann — CHANGES_REQUESTED** [review]

File: src/js/references/components/CreateReference.tsx (hunk 1 above)
Comment (raastinn) [reply src/js/references/components/CreateReference.tsx:24]: No it seem as if the TabsLink api requires an arrow function

**Review by ReeceHoffmann — APPROVED** [review]

**Pull request merged**